### DISCUSSION TITLE
less fps than usual

### DISCUSSION THREAD

**Discussion by shinmorisawa** [ppy/osu#21489] [Q&A]
### Type

Performance

### Bug description

i usually have 240+fps but since today i had like under 60fps.

### Version

2022.1123.0-lazer

### Logs

[database.log](https://github.com/ppy/osu/files/10135410/database.log)
[input.log](https://github.com/ppy/osu/files/10135411/input.log)
[network.log](https://github.com/ppy/osu/files/10135412/network.log)
[performance-draw.log](https://github.com/ppy/osu/files/10135413/performance-draw.log)
[performance-update.log](https://github.com/ppy/osu/files/10135414/performance-update.log)
[performance.log](https://github.com/ppy/osu/files/10135415/performance.log)
[runtime.log](https://github.com/ppy/osu/files/10135416/runtime.log)

**Comment by bdach** [comment]
You will have to provide more context as to what might have changed. Possible causes include

- game update installed
- driver update installed
- OS update installed
- device unplugged from AC power (if laptop)

etc etc. We cannot tell which one it is from logs alone.

**Reply by bdach** [reply]
Also read https://osu.ppy.sh/wiki/en/Performance_troubleshooting.

**Reply by shinmorisawa** [reply]
i didn't do any of the things listed above.

**Reply by bdach** [reply]
If you want us to be able to diagnose this then you'll have to provide *some* information. So far you said that your fps is bad and you said what you *didn't* do. From this information alone I cannot help you.
